twilight-cache-inmemory/CHANGELOG.md

```diff
@@ -6,20 +6,43 @@
 
 - Address new rust CI errors ([#2285](https://github.com/twilight-rs/twilight/issues/2285))
 - [**breaking**] `Member#joined_at` can sometimes be null ([#2288](https://github.com/twilight-rs/twilight/issues/2288))
+- Resolved some warnings when building docs. ([#2331](https://github.com/twilight-rs/twilight/issues/2331))
+- Clippy 1.78 lints ([#2336](https://github.com/twilight-rs/twilight/issues/2336))
```

Comment:
While technically these had the `fix` prefix, clippy lint fixes should not qualify as bug fixes in the changelog. Move it to chore?